src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -334,4 +328,14 @@ export class VerifyAccountRequestProcessor {
   		return ExperianCheckResults.NO_MATCH;
   	}
   }
+
+  async generateUuid(session: ISessionItem): Promise<string> {
```

Comment:
Sorry very minor but can we make this updateVendorUuid.
The problem with generate is someone may assume this just creates a uuid when in fact it also potentially updates the db